### PR TITLE
[WIP] Adding  user tasks

### PR DESCRIPTION
**Pull request by abrennan89**
Fixes #issue-number or description of the problem the PR solves

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/client/kn-reference-commands.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+ 10.136s Traffic is not yet migrated to the latest revision.
+ 10.175s Ingress has not yet been reconciled.
+ 10.348s Ready to serve.
+
+Service 'hello' updated with latest revision 'hello-dghll-2' and URL:
+http://hello.default.apps-crc.testing
+```
+
+## Knative Eventing kn commands
```
Comment (evankanderson) [inline docs/client/kn-reference-commands.md:54]: Deliberately left empty?

File: docs/serving/gettingstarted/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This guide provides information about getting started with deploying and managing Knative applications, and optional configuration settings that you can modify for your Knative Serving deployment.
```
Comment (evankanderson) [inline docs/serving/gettingstarted/README.md:1]: This seems like a pretty short page. Should this be at the start of `docs/serving/README.md`, or is this part of a reorganization of content?

**Comment by evankanderson** [comment]
Also, a lot of these links seem unhappy (for example linking to `getting-started`, but the directory is `gettingstarted`):

```
----------------------------------------------
---- Checking links in the markdown files ----
----------------------------------------------
---- Fri May 15 15:14:58 PDT 2020
----------------------------------------------
docs/README.md
	ERROR	./serving/getting-started-knative-app.md
		Stat docs/serving/getting-started-knative-app.md: no such file or directory
docs/serving/gettingstarted/README.md
	ERROR	./install/_index.md
		Stat docs/serving/gettingstarted/install/_index.md: no such file or directory
docs/serving/gettingstarted/getting-started-knative-app.md
	ERROR	../install/README.md
		Stat docs/serving/install/README.md: no such file or directory
	ERROR	../serving/samples/hello-world/helloworld-go
		Stat docs/serving/serving/samples/hello-world/helloworld-go: no such file or directory
	ERROR	../serving/using-a-custom-domain.md
		Stat docs/serving/serving/using-a-custom-domain.md: no such file or directory
	ERROR	./tag-resolution.md
		Stat docs/serving/gettingstarted/tag-resolution.md: no such file or directory
```

**Comment by abrennan89** [comment]
Actually working with Roland on CLI stuff so this is probably way outdated. Closing.

**Pull request closed without merging**